repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/library.py

```diff
@@ -186,9 +188,11 @@ def parse_entry(entry):
     action = parse_action(entry['action'])
     in_pkgs = parse_packageset(entry.get('in_packageset') or {})
     out_pkgs = parse_packageset(entry.get('out_packageset') or {})
+    release = parse_release(entry.get('release') or {})
```

Comment:
```suggestion
    to_release = parse_release(entry.get('release') or {})
    from_release = parse_release(entry.get('initial_release') or {})
```
I believe the fields in the JSON are named this unintuitively for historical reasons, but for the sake of readability, I'd named them better in the actor.

Reply:
Done.